sites/svelte-5-preview/src/routes/docs/content/03-appendix/02-breaking-changes.md

```diff
@@ -133,6 +133,25 @@ In Svelte 4 syntax, every property (declared via `export let`) is bindable, mean
 
 Setting the `accessors` option to `true` makes properties of a component directly accessible on the component instance. In runes mode, properties are never accessible on the component instance. You can use component exports instead if you need to expose them.
 
+### `immutable` option is ignored
+
+Setting the `immutable` option has no effect in runes mode. This concept is replaced by how `$state` and its variations work.
+
+### Classes are no longer "auto-reactive"
+
+In Svelte 4, doing the following triggered reactivity:
+
+```svelte
+<script>
+	let foo = new Foo();
+</script>
+
+<button on:click={() => (foo.value = 1)}>{foo.value}</button
+>
+```
+
+The reason is that Svelte statically saw the reassignment and used is a hint to trigger a rerender. In runes mode, just wrapping the class with `$state` doesn't make it reactive. The reason is that reactivity is now runtime-based, and classes are left untouched. To make that code reactive, you instead need to make the `value` on `Foo` `$state`.
```

Comment:
adding a bit more specificity

```suggestion
This is because the Svelte compiler treated the assignment to `foo.value` as an instruction to update anything that referenced `foo`. In Svelte 5, reactivity is determined at runtime rather than compile time, so you should define `value` as a reactive `$state` field on the `Foo` class. Wrapping `new Foo()` with `$state(...)` will have no effect — only vanilla objects and arrays are made deeply reactive.
```